docs/tutorials/jokes.md

```diff
@@ -386,6 +386,14 @@ export default function NewJokeRoute() {
           </label>
         </div>
         <div>
+          {actionData?.formError ? (
+            <p
+              className="form-validation-error"
+              role="alert"
+            >
+              {actionData.formError}
+            </p>
+          ) : null}
```

Comment:
it doesn't appear this route is using `useActionData`

Reply:
Copied it over in too much places apparently 😅🙈